udata_hydra/analysis/csv.py

```diff
@@ -131,6 +133,9 @@ async def analyse_csv(
     table_name = hashlib.md5(url.encode("utf-8")).hexdigest()
     timer.mark("download-file")
 
+    # Update resource status to TO_ANALYSE
+    await Resource.update(resource_id, {"status": "TO_ANALYSE"})
```

Comment:
I think this should happen before `analyse_csv` job even starts.

Reply:
Yes but I think it would make sense not to change the status before `config.CSV_ANALYSIS` is checked to be true. I moved it right after.